"Clarify the 'Existing Projects' section of the TypeScript docs: (#52944)
vercel/next.js: docs/02-app/01-building-your-application/06-configuring/01-typescript.mdx

## Existing Projects

> Add TypeScript to your project by renaming a file to `.ts` / `.tsx`. Run `next dev` and `next build` to automatically install the necessary dependencies and add a `tsconfig.json` file with the recommended config options. Finally, copy the `paths` compiler option from your existing `jsconfig.json` file into the compiler options of the `tsconfig.json` file, and delete the `jsconfig.json` file.

Reviewer: ```suggestion
Add TypeScript to your project by renaming a file to `.ts` / `.tsx`. Run `next dev` and `next build` to automatically install the necessary dependencies and add a `tsconfig.json` file with the recommended config options. 

If you already had a `jsconfig.json` file, copy the `paths` compiler option from the old `jsconfig.json` into the new `tsconfig.json` file, and delete the old `jsconfig.json` file.
```
